### DISCUSSION TITLE
Error upgrading from 2.5.287 to 2.5.294

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#5989] [Error / Bug Report]
I get the following error following the upgrade from 2.5.287 to 2.5.294.

G:\[[pathremoved]]\server\views\page.pug:13 11| block body 12| #root > 13| page( 14| locale=page.localeCode 15| path=page.path 16| title=page.title The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined

What other information can I provide to help troubleshoot this error?
I reverted back to 2.5.294 to get back up an running for now.

**Comment by EricFromCanada** [comment]
Sometimes it's necessary to replace the `wiki/` folder entirely when performing an upgrade, as [recommended by the docs](https://docs.requarks.io/install/upgrade).

**Comment by ghost** [comment]
Thank you for getting back to me.  I have upgraded wiki.js many times and I always follow the 5 PowerShell in-place upgrade steps you mentioned.  This time it's just not working.  I tried again today this time with 2.5.295, and fully deleted the current directory (saved the config) before starting.  

I got the same exact error again.  What else can I provide to help get to the bottom of this?  Thank you very much.

**Comment by ghost** [comment]
I realized my issue.  I have an NSSM service that keeps wiki.js running, and this was causing some caching issues.  Thanks for an amazing product, sorry for the dumb problem.
